### PR TITLE
 python3Packages.svglib: init at 1.0.0 

### PR DESCRIPTION
**Pull request by trepetti**
###### Motivation for this change

[svglib](https://github.com/deeplook/svglib) is a pure Python package for manipulating and exporting SVGs. Due to its focus on portability, it has already seen use in [an academic molecular dynamics package](https://github.com/gayverjr/pyemap) and a tool for generating gcode from PCB Gerber files for CNC fabrication. The latter of which, [FlatCAM](https://bitbucket.org/jpcgt/flatcam/src/master/), I am in the process of packaging, and for which I am submitting this last remaining requirement as a stepping stone.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -8416,6 +8416,12 @@
     githubId = 207457;
     name = "Matthieu Chevrier";
   };
+  trepetti = {
+    email = "trepetti@cs.columbia.edu";
+    github = "trepetti";
+    githubId = 25440339;
+    name = "Tom Repetti";
+  };
```
Comment (jonringer) [inline maintainers/maintainer-list.nix:8424]: Please make this into a separate commit:

```
maintainers: add <name>
```

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -6550,6 +6550,8 @@ in {
 
   stevedore = callPackage ../development/python-modules/stevedore {};
 
+  svglib = callPackage ../development/python-modules/svglib {};
```
Comment (jonringer) [inline pkgs/top-level/python-packages.nix:6553]:
```suggestion
  svglib = callPackage ../development/python-modules/svglib { };
```

**Comment by jonringer** [comment]
should be only two commits:
```
maintainers: add <name>
python3Packages.svglib: init at 1.0.0
```

File: maintainers/maintainer-list.nix (hunk 1 above)
Comment (trepetti) [reply maintainers/maintainer-list.nix:8424]: Done. Thanks for your feedback.

File: pkgs/top-level/python-packages.nix (hunk 2 above)
Comment (trepetti) [reply pkgs/top-level/python-packages.nix]: Whoops, yeah, the package above had the same stylistic issue. Lesson learned!

**Comment by trepetti** [comment]
> should be only two commits:
> 
> ```
> maintainers: add <name>
> python3Packages.svglib: init at 1.0.0
> ```

After an initial rebase misfire, I think these two reorganized commits should do the trick. Thank you for your review.

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 95632 [1](https://github.com/Mic92/nixpkgs-review)<details>  <summary>2 packages built:</summary><br>- python37Packages.svglib<br>- python38Packages.svglib</details>
